plugins/osdn/ovs/controller.go

```diff
@@ -187,6 +213,10 @@ func (c *FlowController) Setup(localSubnetCIDR, clusterNetworkCIDR, servicesNetw
 	otx.AddPort(TUN, 2, "type=internal")
 	otx.AddPort(VOVSBR, 3)
 
+	// Table 253: rule version stamp; note action is hex bytes separated by '.'
+	pluginStamp := getPluginStamp(c.multitenant)
+	otx.AddFlow("%s, %s%s.%s", STAMP_TABLE, STAMP_ACTION, pluginStamp[0], pluginStamp[1])
+
```

Comment:
Can we add this flow rule at the end of the Setup()?
For some reason, tun0/docker0 transaction or net sysctl fails (these are executed after br0 transaction), we won't be able to fix setup by restarting the openshift service as alreadySetup() will return true.